KsApi/GraphSchema.swift

```diff
@@ -802,3 +848,9 @@ extension Query.Notifications {
     hasher.combine(self.description)
   }
 }
+
+// MARK: - Schema Constants
+
+public extension Query {
+  static let defaultPaginationCount: Int = 25
+}
```

Comment:
I didn't fully read this file to understand it, (as it would take too long to understand this graph schema) but it seems pretty good in that it works.